.github/workflows/build.yml

```
@@ -40,7 +40,7 @@ jobs:
       - name: Disable Spotlight
         run: sudo mdutil -a -i off
       - name: Run xcode-select
-        run: sudo xcode-select --switch /Applications/Xcode_14.3.1.app
+        run: sudo xcode-select --switch /Applications/Xcode_15.0.1.app
       - name: Install dependencies
         run: brew install coreutils ninja
```

I've been playing around with fixing the python environment for the ARM builds, and it seems like installing a newer setuptools gets it working, but only when installed through pip and not homebrew. No idea why, but something like this should do it without the need for a new patch.

```suggestion
        # Remove pip call after Chromium 119
        run: |
          brew install coreutils ninja
          pip3 install -U setuptools
```

The build script that needs setuptools should only run once during the early build setup, so this only needs to be in the first step and not copied to all the others.